packages/trader/src/Stores/Modules/Statement/Helpers/format-response.js

```diff
@@ -32,5 +33,6 @@ export const formatStatementTransaction = (transaction, currency, active_symbols
         action_type: transaction.action_type,
         purchase_time: transaction.purchase_time,
         transaction_time: transaction.transaction_time,
+        ...(withdrawal_details && { withdrawal_details }),
```

Comment:
Can we just use `transaction.withdrawal_details` here?

```suggestion
       withdrawal_details: transaction. withdrawal_details,
```